composer/yamls/models/bert-base.yaml

```diff
@@ -52,14 +52,11 @@ max_duration: 256_000_000
 train_batch_size: 4000
 eval_batch_size: 2000
 seed: 19
-device:
```

Comment:
why are `device` and `precision` removed?

Reply:
> Removed the device and precision settings from the YAMLs, as the trainer now uses smart defaults (AMP + GPU if cuda is available; otherwise FP32 + CPU). This is necessarry as AutoYAHP will parse the yamls into the DeviceGPU() / DeviceCPU() instances directly, and DeviceGPU() throws an error if cuda is not available (before, yahp would parse into DeviceGPUHparams(), which could then be replaced with DeviceCPUHparams()). (While it would have also worked to patch the yaml directly, I thought it'd be generally better to make the yamls more portable).